### PR TITLE
add oro_prec recipe

### PR DESCRIPTION
**Pull request by rlange2**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rlange2** [comment]
I'm note sure about `certification verify failed` in Appveyor, however, the PR is ready for review.

File: recipes/orographic-precipitation/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - numpy
```
Comment (marcelotrevisani) [inline recipes/orographic-precipitation/meta.yaml]: should you add ``matplotlib`` as well here?
```suggestion
    - numpy
    - matplotlib
```

**Comment by marcelotrevisani** [comment]
> I'm note sure about `certification verify failed` in Appveyor, however, the PR is ready for review.

don't worry about it, you can ignore appveyor

File: recipes/orographic-precipitation/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{% set name = "orographic_precipitation" %}
+{% set version = "0.1rc0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "13559a0f324f9761ee2534430feb628f71416ff1c3b10c79d3452b2c276246b2"
```
Comment (marcelotrevisani) [inline recipes/orographic-precipitation/meta.yaml]:
```suggestion
  sha256: 13559a0f324f9761ee2534430feb628f71416ff1c3b10c79d3452b2c276246b2
```

Could you please remove the quotes from ``sha256``?

Comment (rlange2) [reply recipes/orographic-precipitation/meta.yaml]: Yes, of course. I'm just starting to familiarise myself with yaml-files. This one came up when I executed:
`conda skeleton pypi your_package_name`

Should I do the same with the quotation marks in the `url` or should I leave them be?

File: recipes/orographic-precipitation/meta.yaml (hunk 1 above)
Comment (rlange2) [reply recipes/orographic-precipitation/meta.yaml]: That's fine. I included some plotting function at first but that is not really part of the package but rather helps visualising the example in my README. So I excluded the function.

File: recipes/orographic-precipitation/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [reply recipes/orographic-precipitation/meta.yaml]: We are developing a new tool with the intention to replace the current conda skeleton
it is not ready yet, I didn't generate version for it yet. But if you want to, you can give a try
https://github.com/marcelotrevisani/grayskull

And I thought I had fixed this problem regarding the quotes on ``sha256`` on conda skeleton
which version are you using of ``conda-build``?

the ``url`` is fine, you can let it there, there is no need to change, just if you want to

Comment (rlange2) [reply recipes/orographic-precipitation/meta.yaml]: > We are developing a new tool with the intention to replace the current conda skeleton
> it is not ready yet, I didn't generate version for it yet. But if you want to, you can give a try
> https://github.com/marcelotrevisani/grayskull

Certainly, I want to give it a try. Do you mind giving me a rundown on how to use it? Do I create an instance from `AbstractRecipeModel` in my git directory, supply the package name and version number? Sorry for the inconvenience, this is still very new to me.

> And I thought I had fixed this problem regarding the quotes on `sha256` on conda skeleton
> which version are you using of `conda-build`?

`conda-build 3.18.9`
Maybe you did and it's simply an issue on my end.

> the `url` is fine, you can let it there, there is no need to change, just if you want to

Thank you for your help and the approval!

**Pull request merged**